### ISSUE TITLE
emphasis on the relative nature of support in Star voting

### ISSUE THREAD

**Issue by wolftune** [nardo/Equal.Vote#27]
I think one of the main knee-jerk responses to star voting are that people have different ideas of what 5-stars means or 4-stars. We need a really good page on equal.vote that not only explains that zero-support to max-support is fine, universal, and *relative* to the pool of candidates in an election.

The page should include educational suggestions & plans in order to convince readers that we will succeed at getting across this interpretation. So, we need to make it clear that we're not relying on people *necessarily* getting this right immediately, but that we have specifically worked out how to make sure everyone gets it.

We should even acknowledge that the relative nature does mean that adding extreme candidates (or removing them) can cause a shift in the relative meaning of the scores, and that this is just an unfortunate fact but one that doesn't undermine Star voting's advantages.

Again, the point is to be a page to send to those people who say "scoring doesn't work, we don't agree about what 5-stars mean, cultural differences etc"

**Comment by nardo** [comment]
Most of the pushback I've seen on this front comes when the range is larger
- like 0-9.

But sure, a page describing it - or even a blog post - would be good.

**Comment by endolith** [comment]
and point out that rankings are even _worse_ for comparing between people.  

If I hate both candidates but I hate B more, and you love both candidates but love A more, we will both vote A > B, though they mean very different things.  

Likewise, if I slightly prefer A over B, and you greatly prefer A over B, we both vote A > B, but they mean very different things.


> In summary, one may state that scores originally obtained as ipsative measures may legitimately be employed only for purposes of intraindividual comparisons. Normative measures may be employed for either interindividual or intraindividual comparisons. - Hicks, Lou E. 1970. “Some Properties of Ipsative, Normative and Forced-Choice Normative Measures.” Psychological Bulletin 74(3):167-184.


Also could be answered here: [Politics.SE: How does range voting deal with lack of objective scoring?](https://politics.stackexchange.com/q/652/10373)

**Comment by wolftune** [comment]
I've seen the pushback strengthened when we compare to Yelp and Amazon etc. or even Olympics, because those are all more absolute. Instead of relative support, it's bad to great.

So, we need to make this contrast clear, embrace it and not hide from it. Something like: "While a 1-star review on Amazon means a bad product and 5-stars means a great product, and there could be just one product in question or ten thousand, and maybe nothing deserves 5-stars or maybe they all deserve 5-stars. By contrast, Star voting focuses on which candidate to elect from a fixed pool of candidates. All the candidates can be pretty great, but if you still have real preferences, you will give no support (i.e. zero stars) to the candidate(s) you least prefer. Or if you feel negative about all the candidates but still have real preferences, you will still give full support (5 stars) to the candidate(s) you prefer most. If you *don't* have any strong preferences, you could always score all the candidates low or all high. While that could perhaps be seen as a general no-confidence vote or an affirmation of the candidates, it will mean your vote just makes no difference to the outcome (which is fine, since you have no preferences)."

Something of this where we emphasize the relative issue, note that things still work fine even if people do not understand the way this is unlike Yelp, and that we will have clear educational materials for this.

I'd also emphasize that experience with Star voting will only improve people's understandings. If voters are unhappy with a result when they gave nobody full support, they will understand later that they have to give full support to their favorite(s).

I want to emphasize the plural stuff (as I've done here) to make it clear that it's fine to have multiple favorites or least favorites who all get full or no support.

I think the simplest elevator pitch that should be included at the top of the page I'm proposing would give an example election: "On a scale from Donald Trump to Rand Paul, how would you score Marco Rubio?" The "on-a-scale from worst-candidate to best-candidate" is the point. Another way to put this is:

* pick one candidate you prefer as much as any other, that's a 5
* pick one candidate you prefer to no others, that's a 0"
* now you have your scale, score the others on that scale (others can be also 0's or 5's if appropriate)
* *optional optimization*: ignore any candidates you believe have no realistic chance of winning when you consider the scale for this candidate pool, but then you can go ahead and score those candidates anyway
    * in other words: make the scale from 0-5 be the range among the viable candidates, and any non-viable candidate who is off the scale will just get a 0 or a 5 depending on being off-the-scale bad vs off-the-scale good.

That last bit is a bit convoluted, should be relegated to a footnote or further-down-the-page, but it addresses the issue of introducing non-viable extremes in order to change the anchor points.

In the end, this is tied to having such superb educational material that people see how the roll-out will be successful, voters will understand, have realistic expectations of the outcomes, etc. because the outreach and education was so well done…

**Comment by endolith** [comment]
>  While that could perhaps be seen as a general no-confidence vote or an affirmation of the candidates, it will mean your vote just makes no difference to the outcome (which is fine, since you have no preferences)."

Actually I think this could be marketed as an advantage of cardinal systems: If you hate all of the candidates, you can say so on the ballot and your "0" votes will be counted and shown in the totals.  Increases turnout, more accurate representation of voter opinions, etc.

https://electology.org/sites/default/files/9-way%20Honest%20Assessments%20of%20candidates.png

**Comment by wolftune** [comment]
@endolith yes, I agree, and I want the results to report the difference between candidates with lots of marked-zeros vs unscored, but there's evidence that some people will mark a zero just when they are unfamiliar with a candidate. Anyway, the point is that the ability of score to give everyone low scores (which rank ballots can't do without a none-of-these or none-of-the-others option) is both nice *and* plays into the concerns about the scores being somehow absolute. So, we need to *not* emphasize this because we *prefer* voters to normalize their scores in most cases and to understand scores as normalized to the candidate pool. It's just that we don't want to suggest that Star voting *requires* this normalize understanding in order to work okay.
